src/locutus/api/terminology_mapping.py

```diff
@@ -22,18 +22,26 @@ def get(self, id, code):
 
         # We should recieve a dictionary with a single key
         for coding in mappings[code]:
-            response["mappings"].append(coding.to_dict())
+            if not hasattr(coding, 'valid') or coding.valid:
+                response["mappings"].append({
+                            "code": coding.code,
+                            "display": coding.display,
+                            "system": coding.system,
+                            "valid": coding.valid if hasattr(coding, 'valid') else None

```

Comment:
We didn't really discuss whether the valid flag should be passed on to client applications. If we operate under a stricter FHIR oriented perspective, they should not. However, it may make sense to expand our view of this.

Reply:
I never thought about leaving it out. That makes sense to me. Especially since I'm only returning valid mappings anyway. I'll revert most of this since it can use 'coding' again.